kernel/kata_config_version

```diff
@@ -1 +1 @@
-25
+26
```

Comment:
Hi @wenlxie - sorry about this, but I think the kernel config version has moved ahead in the master branch again - can you rebase and repush... sorry for the hassle - let's see if we can get this one moving again...